tools/codegen/gen.py

```diff
@@ -429,6 +430,62 @@ def generate_defn(faithful: bool) -> str:
     return at::_ops::{f.func.name.unambiguous_name()}::redispatch({exprs_str});
 }}
 """
+
+        result = generate_defn(False)
+        if sig_group.faithful_signature is not None:
+            result += generate_defn(True)
+
+        return result
+
+
+# Generates Static.cpp.
+# This generates static unboxing wrapper for ATen ops.
+@dataclass(frozen=True)
+class ComputeStaticUnboxingWrapper:
+
+    @method_with_native_function
+    def __call__(self, f: NativeFunction) -> Optional[str]:
+        # We unconditionally generate function wrappers,
+        sig_group = CppSignatureGroup.from_native_function(f, method=False, fallback_binding=f.manual_cpp_binding)
+
+        def generate_defn(faithful: bool) -> str:
+            if faithful:
+                sig = sig_group.faithful_signature
+                assert sig is not None
+            else:
+                sig = sig_group.signature
+            # gather all the arguments from native function, including "out"
+            args = f.func.arguments.flat_non_out + (list(f.func.arguments.out) if f.func.arguments.out else [])
+
+            # parse arguments into C++ code
+            arguments = unboxing.convert_arguments(args, f.func.arguments.tensor_options)
+
+            # for each C++ argument, generate the conversion code
+            code_connector = "\n\t\t"
+            code_list = []
+            for k in sig.arguments():
+                code_list.extend(arguments[k.name].code)
+            code = code_connector.join(code_list)
+
+            # function call and push back to stack
+            func_call_and_push = unboxing.generate_unboxed_kernel_call(f, sig, arguments)
+
+            # escape double quote in schema, get rid of extra double quotes
+            schema = json.dumps(sig.func.__str__())[1:-1]
+
+            return f"""
+OperatorGenerator(
```

Comment:
What does `OperatorGenerator` do?

I thought that instead of this, we'd want to define an ordinary function like `at::unboxing::{op_name}` for every op that can be called directly.